### DISCUSSION TITLE
How can I save my file to local storage?

### DISCUSSION THREAD

**Discussion by EuRo47** [vhqtvn/VHSCode-Android#190] [Q&A]
Hello! Could you tell me how can I actually save file either local tablet storage or card. 
App has access to memory. But still, when I was trying to save in local, massege "Opening locals folders in Unsupported. Link to screenshot:
https://ibb.co/NSYWb7n
I would like to code in your app, using tablet. It is so comfortable.

**Comment by vhqtvn** [comment]
Does this occur in latest version?

**Reply by ghost** [reply]
Yes it does![Screenshot_20221228_221843.jpg](https://user-images.githubusercontent.com/96946509/209845382-6af5fde1-7351-4fb0-99e9-edc9f0f79808.jpg)
